build.jam

```diff
@@ -0,0 +1,60 @@
+# Copyright René Ferdinand Rivera Morell 2024
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+import-search /boost/config/checks ;
+import config : requires ;
+
+project /boost/geometry
+    : common-requirements
+        <library>/boost/algorithm//boost_algorithm
+        <library>/boost/any//boost_any
+        <library>/boost/array//boost_array
+        <library>/boost/assert//boost_assert
+        <library>/boost/concept_check//boost_concept_check
+        <library>/boost/config//boost_config
+        <library>/boost/container//boost_container
+        <library>/boost/core//boost_core
+        <library>/boost/endian//boost_endian
+        <library>/boost/function_types//boost_function_types
+        <library>/boost/fusion//boost_fusion
+        <library>/boost/integer//boost_integer
+        <library>/boost/iterator//boost_iterator
+        <library>/boost/lexical_cast//boost_lexical_cast
+        <library>/boost/math//boost_math
+        <library>/boost/mpl//boost_mpl
+        <library>/boost/multiprecision//boost_multiprecision
+        <library>/boost/numeric_conversion//boost_numeric_conversion
+        <library>/boost/polygon//boost_polygon
```

Comment:
Is it possible to distinguish between real dependencies (the library uses it)
and other dependencies?
Other depencie scan be:
* optional adaptations
* dependency for test only
* dependency for examples only

Reply:
Probably not that easy and this is probably generated.
I'm doubting how we should organize our library such that this is more clear.

For the test, it looks good to me, thanks for your efforts!

Reply:
All right, it's fine for now, you made already a huge effort in all libraries. Thanks again, great work.
We will look at it ourselves later.